docs/welcome/welcome-to-elrond.md

```diff
@@ -9,11 +9,11 @@ title: Welcome to Elrond
 - **When:** Mainnet live, project started in 2018
 - **How:** Scales via Adaptive State Sharding, consensus via Secure Proof of Stake
 - **Highlights:**
-  - Performance: 15.000 current TPS, 6s block time, $0.001 tx cost, can scale beyond 100k (263k TPS in testnet)
+  - Performance: 15.000 current TPS, 6s block time, $0.001 tx cost, can scale beyond 100k (263,000 TPS in testnet)
```

Comment:
You can also change 15.000 -> 15,000 and 100k -> 100,000

Reply:
```suggestion
  - Performance: 15,000 current TPS, 6s block time, $0.001 tx cost, up to 100,000 (263,000 TPS in testnet)
```